pkg/server/api/bundle/v1/service.go

```diff
@@ -162,6 +191,10 @@ func (s *Service) PublishJWTAuthority(ctx context.Context, req *bundlev1.Publish
 
 // ListFederatedBundles returns an optionally paginated list of federated bundles.
 func (s *Service) ListFederatedBundles(ctx context.Context, req *bundlev1.ListFederatedBundlesRequest) (*bundlev1.ListFederatedBundlesResponse, error) {
+	rpccontext.AddRPCAuditFields(ctx, logrus.Fields{
+		"page_size":  req.PageSize,
+		"page_token": req.PageToken,
```

Comment:
Is this information useful to an auditor? I think they are only significant if you know the exact size and ordering of the data at the time the request was issued

Reply:
that is the only information in request, I dont think it is useful either.
about size it is for response, and ordering is defined from datastore, 
what do you think about removing page_size and token, but add response size?
something like:
```
fields["respone_size"] = len(resp.Bundles)
```
or maybe we can keep token and add next token:
```
fields["page_token"] = req.PageToken
fields["respone_size"] = len(resp.Bundles)
fields["next_page_token"] = resp.NextPageToken()
```

Reply:
Echoing what @evan2645 said, I'm not sure any of these request parameters are really all that useful from an auditing perspective. There are no restrictions over what data is exposed over the `ListFederatedBundles` API, e.g. client `A` sees bundles `1` and `2` whereas client `B` sees bundles `3` and `4` in the response. So if this RPC is called, the assumption would be that the client received all the bundles.

Whether the client intentionally ignores any of the bundles during its handling of the paginated results (i.e. there are 3 pages worth of data, but the client only requests pages `1` and `2`) doesn't seem necessary to log when we may not be able to receive the same data from the server when replaying the request(s) with the audited request parameters.